Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -194,6 +197,7 @@ class NavigationHeader extends React.Component<DefaultProps, Props, any> {
         style={[
           styles[name],
           styleInterpolator(props),
+          subviewStyle[name],
```

Comment:
I'm confused about the structure of the `subviewStyle` object. How does it look? Can you give an example?